src/components/Portfolio/LoanList.vue

```diff
@@ -0,0 +1,245 @@
+									</div>
+									<div class="tw-mb-1 tw-text-secondary">
+										{{ formatDate(loan.terms.disbursalDate) || '(Endpoint TBD)' }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div>
+										${{ loan.terms.loanAmount }}
+									</div>
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									{{ loan.lenderRepaymentTerm || '-' }} mos
+								</div>
+							</td>
+							<td class="tw-text-right tw-px-2">
+								<div>
+									<div class="tw-mb-1">
+										${{ loan.userProperties.loanBalance.amountRepaidToLender }} repaid to you
+									</div>
+								</div>
+							</td>
+							<td class="tw-max-w-[150px] tw-whitespace-normal tw-break-words tw-px-2">
+								<div class="tw-items-center">
+									<template v-if="loan.teams?.values?.[0]">
+										<img
+											v-if="loan.teams.values[0].image?.url"
+											:src="loan.teams.values[0].image.url"
+											:alt="`${loan.teams.values[0].name} team image`"
+											class="tw-w-5 tw-h-5"
+										>
+										<span>{{ loan.teams.values[0].name }}</span>
+									</template>
+									<span v-else>-</span>
+								</div>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
+			<div class="scroll-gradient"></div>
+		</div>
+	</div>
+</template>
+
+<script>
+import { KvFlag, KvLoadingPlaceholder } from '@kiva/kv-components';
+import {
+	DEFAULTED,
+	ENDED,
+	EXPIRED,
+	FUNDED,
+	FUNDRAISING,
+	PAYING_BACK,
+	RAISED,
+	REFUNDED,
+} from '#src/api/fixtures/LoanStatusEnum';
+
+const DELINQUENT = 'payingBackDelinquent';
+
+export default {
+	name: 'LoanList',
+	inject: ['cookieStore'],
+	props: {
+		loans: {
+			type: Array,
+			default: () => []
+		},
+		loading: {
+			type: Boolean,
+			default: true
+		}
+	},
+	components: {
+		KvFlag,
+		KvLoadingPlaceholder
+	},
+	methods: {
+		formatDate(date) {
+			if (!date) return '';
+			return new Date(date).toLocaleDateString('en-US', {
+				month: 'short',
+				day: 'numeric',
+				year: 'numeric'
+			});
+		},
+		getPrintableStatus(rawStatus) {
+			const mapping = {
+				[FUNDRAISING]: 'Fundraising',
+				[FUNDED]: 'Funded',
+				[RAISED]: 'Raised',
+				[PAYING_BACK]: 'Paying Back',
+				[ENDED]: 'Ended',
+				[DELINQUENT]: 'Delinquent',
+				[DEFAULTED]: 'Defaulted',
+				[REFUNDED]: 'Refunded',
+				[EXPIRED]: 'Expired'
+			};
+			return mapping[rawStatus] || rawStatus;
+		},
+	},
+	computed: {
+		getPartnerUrl() {
+			return partnerId => `/about/where-kiva-works/partners/${partnerId}`;
+		}
+	},
+	data() {
+		return {
+			placeholders: [
+				{ span: 4 },
+				{ span: 1 },
+				{ span: 1, marginLeft: true },
+				{ span: 1, marginLeft: true },
+				{ span: 1, marginLeft: true },
+				{ span: 2, marginLeft: true },
+				{ span: 2 }
+			]
+		};
+	}
+};
+</script>
+
+<style scoped>
+.scroll-gradient {
+	pointer-events: none;
+	position: absolute;
```

Comment:
A lot of these can be replaced with TW classes, if you add `lang="postcss"` to the `style` tag

Reply:
Just did what I think you mean by this, lmk if anything looks off

Reply:
Yup that's what I meant, thanks.